test/mlp_l4_mnist/fully-connected-layer.cpp

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Why is this file being added ?

Reply:
Because this file takes the file name as input. We have a script(run-mlp-benchmark) that directly copies this file to the compiler.  Same thing for the next comment. I can add the default values for this. 

In the previous version of compiler, user has to manually change the file name so that the generated files don't get overlap. Here we can just change the names in our script.